src/components/Header.js

```diff
@@ -3,9 +3,27 @@ import React from "react";
 const Header = () => {
   return (
     <div>
-      <h1> React and Morty </h1>
+      <HeaderStyles> React and Morty </HeaderStyles>
     </div>
   );
 };
 
+
+
+const HeaderStyles = styles.h1`
+
+height: 75px;
+justify-content: cebter;
+margin: 5px 5px 5px 5px; 
```

Comment:
`margin: 5px` should work the same as `margin: 5px 5px 5px 5px` :)

Reply:
This goes back a couple weeks. Thanks for the reminder.